src/main/java/gregtech/common/blocks/BlockOre.java

```diff
@@ -181,14 +182,7 @@ public IBlockState getOreBlock(StoneType stoneType) {
 
     @Override
     @SideOnly(Side.CLIENT)
-    public void onTextureStitch(TextureStitchEvent.Pre event) {
-        event.getMap().registerSprite(MaterialIconType.block.getBlockPath(material.getMaterialIconSet()));
-        for (IBlockState state : this.getBlockState().getValidStates()) {
-            StoneType stoneType = state.getValue(STONE_TYPE);
-            event.getMap().registerSprite(stoneType.backgroundTopTexture);
-            event.getMap().registerSprite(stoneType.backgroundSideTexture);
-        }
-    }
+    public void onTextureStitch(TextureStitchEvent.Pre event) { }
```

Comment:
Is this needed any more, since it is an empty override?

Reply:
There because of the interface, its not `default`